portfolio/src/main/java/com/google/sps/servlets/NewTaskServlet.java

```diff
@@ -18,18 +18,20 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String name = Jsoup.clean(request.getParameter("name"), Whitelist.none());
+    String email  = Jsoup.clean(request.getParameter("email"), Whitelist.none());
     String title = Jsoup.clean(request.getParameter("title"), Whitelist.none());
-    long timestamp = System.currentTimeMillis();
-
+    
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Task");
     FullEntity taskEntity =
         Entity.newBuilder(keyFactory.newKey())
+            .set("name", name)
+            .set("email", email)
             .set("title", title)
-            .set("timestamp", timestamp)
             .build();
     datastore.put(taskEntity);
-
+    
```

Comment:
This line contains spaces with nothing after. We call that "trailing whitespace" and it should be avoided for style purposes. Please remove the extra spaces